ee/clickhouse/models/session_recording_event.py

```diff
@@ -49,3 +52,7 @@ def create_session_recording_event(
         capture_exception(Exception(f"Session recording event data too large - {len(snapshot_data_json)}"))
 
     return str(uuid)
+
+
+def update_recordings_ttl(weeks: int) -> None:
```

Comment:
Nit: IMO the 3-file-split is not justified here, just adds unneeded indirection.